R/replace_negatives.R

```diff
@@ -0,0 +1,35 @@
+#' Replace negative incidence with imputed data and redistribute
+#' residuals across observations before adjustment date
+#'
+#' @param data data frame
+#' @param measure death or case
+#' @return data frame after replacement
+#'
+#' @export
+#'
+replace_negatives <- function(data, measure) {
+
+  # find observations with negative inc
+  adjustments <- covidData::get_negative_cases(data)
+
+  # replace negative incidents with imputed data for all replated locations
+  for (i in 1:nrow(adjustments)) {
```

Comment:
We probably need `seq_len(nrow(adjustments))` here in case there are 0 adjustments to make (i.e., no observations of negative incidence)?  If this is a bug, we should make sure we have a unit test that detects it.

```suggestion
  for (i in seq_len(nrow(adjustments))) {
```